.github/actions/check-logs/action.yml

```diff
@@ -0,0 +1,41 @@
+name: Check logs
+description: Parses logs for crashes and failures
+
+inputs:
+  logs:
+    description: The path to load GTest XML from.
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - name: Parse logs
+      env:
+        logs: ${{ inputs.logs }}
+      run: |
+        import os
+        import sys
+        import glob
+
+        def error(title, file, line, msg):
+            print(f"::error file={file},line={line},title={title}::{msg}")
+
+        input_glob = "./.github/actions/check-logs/temp/*.log"
+        # input_glob = os.environ.get('logs')
+        log_files = glob.glob(input_glob)
+
+        crashes = ['SIGSEGV', 'SIGFPE', 'SIGILL', 'SIGABRT', 'SIGBUS', 'Interrupt signal']
```

Comment:
>'Interrupt signal' [](http://example.com/codeflow?start=71&length=18)

This is not a crash. It's a OSConfig termination by Ctrl +C when console app for example, or stop service.